defaults/main.yml

```diff
@@ -9,6 +9,12 @@ omero_server_ice_version: "3.6"
 # Use a virtualenv
 omero_server_virtualenv: false
 
+# Python 3?
+omero_server_python3: false
+
+# Make bin/omero a wrapper script for the virtualenv omero
+omero_server_python3_replace_omero: true
```

Comment:
I imagine we can eventually remove this flag by making bin/omero work in either case.

Reply:
* `replace: true` prevents `python bin/omero` usage.
* `replace: false` prevents Python services from starting.